### DISCUSSION TITLE
podman desktop for windows relay on WSL2, but it bugs lots

### DISCUSSION THREAD

**Discussion by H20230103** [podman-desktop/podman-desktop#4991] [General]
As a long times Docker desktop user, i am interest with podman desktop.
I see the podman desktop for windows, need WSL 2 install first.
but it has lots bugs.[wsl2 is ugly]

which docker desktop also provide hyper-v  version.
It more stable and reliable than WSL2

for many bugs of WSL2, you may check:

https://github.com/microsoft/WSL/issues?q=is%3Aissue+is%3Aopen+error+code

Microsoft won't fixes these wsl2 bugs.

**Comment by jeffmaury** [comment]
Hyper-V implementation is available as Tech Preview since 4.8.0-rc1.
Integration within Podman Desktop will come once it's out of Tech Preview

**Reply by jcralbino** [reply]
Hello
I want to install podman in a physical windows host without requiring wsl2

Is the hyper-v the only option ? Can the tech-preview you mention be available as a airgap full download
